test/rules/noShadowedVariableRuleTests.ts

```diff
@@ -0,0 +1,44 @@
+/*
+ * Copyright 2013 Palantir Technologies, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+describe("<no-shadowed-variable>", () => {
+    it("ensures that variable declarations are not shadowed", () => {
+        var fileName = "rules/no-shadowed-variable.test.ts";
+        var NoShadowedVariableRule = Lint.Test.getRule("no-shadowed-variable");
+        var failureString = NoShadowedVariableRule.FAILURE_STRING + "shadowed'";
```

Comment:
if you find the problem, please open a new PR to fix it. I'll take care of the small fixes necessary in this PR.